operator/v1/types_network.go

```diff
@@ -100,12 +100,79 @@ type DefaultNetworkDefinition struct {
 	KuryrConfig *KuryrConfig `json:"kuryrConfig,omitempty"`
 }
 
+// SimpleMacvlanConfig contains configurations for macvlan interface.
+type SimpleMacvlanConfig struct {
+	// master is the host interface to create the macvlan interface from.
+	// If not specified, it will be default route interface
+	// +optional
+	Master string `json:"master,omitempty"`
+
+	// IPAMConfig configures IPAM module will be used for IP Address Management(IPAM).
+	IPAMConfig IPAMConfig `json:"ipamConfig"`
+
+	// mode is the macvlan mode: bridge, private, vepa, passthru. The default is bridge
+	// +optional
+	Mode MacvlanMode `json:"mode,omitempty"`
+
+	// mtu is the mtu to use for the macvlan interface. if unset, host's
+	// kernel will select the value.
+	// +optional
+	MTU uint32 `json:"mtu,omitempty"`
+}
+
+// StaticIPAMAddresses provides IP address/Gateway pairs for static IPAM addresses
+type StaticIPAMAddresses struct {
+	// Address is the ip address in CIDR
+	Address string `json:"address,omitempty"`
```

Comment:
Please fix "`ip`" → "`IP`" in all comments.
Also, not clear what "in CIDR" means here. I think you mean "in CIDR format"?
Also, this is required isn't it? (not `omitempty`)

Reply:
Changed. Address should be `omitempty` because address will be given as pod annotation as https://github.com/intel/multus-cni/issues/278#issuecomment-506222323
In this case, address in net-attach-def is empty.

Reply:
Change 'ip' to 'IP'.